package-lock.json

```diff
@@ -24212,4 +24212,4 @@
       }
     }
   }
-}
+}
```

Comment:
Fix this. There is and extra parenthesis.

Reply:
Yes, I restored the original package-lock.json